README.md

```diff
@@ -33,9 +33,9 @@ and to your application module `build.gradle`:
 ```groovy
 dependencies {
     // use this for the basic c3media version
-    compile "info.metadude.kotlin.library.c3media:base:${version}"
```

Comment:
Rebase interactive your changes into the original commits - don't put the changes on top of the branch.